jcheck/src/main/java/org/openjdk/skara/jcheck/IssuesTitleCheck.java

```diff
@@ -28,10 +28,15 @@
 
 import java.util.ArrayList;
 import java.util.Iterator;
+import java.util.List;
 import java.util.logging.Logger;
+import java.util.regex.Pattern;
 
 public class IssuesTitleCheck extends CommitCheck {
     private final Logger log = Logger.getLogger("org.openjdk.skara.jcheck.issuesTitle");
+    private final static List<String> VALID_WORD_WITH_TRAILING_PERIOD = List.of("et al.", "etc.", "...");
+    private final static List<String> EXECUTABLE_NAMES = List.of("javac", "dpkg");
```

Comment:
This list would potentially get very long. I'm not sure it's worth trying to maintain one. The other patterns seem ok to me.

Reply:
I was thinking we should ask if David Holmes has any thoughts about this list...

Reply:
No I think it should just be removed. Otherwise we will end up having to list every executable in the JDK as well as basically every build tool used when building it, and even so, that would be a very JDK project specific list of executables. Several of those would also overlap with words that we would want to flag, e.g. "make", "java".

Reply:
Honestly, while I appreciate the attempt to improve this, I don't see what the rules are trying to achieve. This is not a grammatical context, and the no-period rule means this is not expected to be a sentence; and we are not trying to enforce title-case so ... what exactly does "Should start with a capital" achieve - especially with all the noted, if not programmed, exceptions?

Reply:
@dholmes-ora There are more details in the issue description. https://bugs.openjdk.org/browse/SKARA-2248